src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
also change Props to something more descriptive, e.g. "MovieCardProps"